crates/turborepo-lib/src/shim/parser.rs

```diff
@@ -301,6 +328,7 @@ mod test {
         pub color: bool,
         pub no_color: bool,
         pub relative_cwd: Option<&'static [&'static str]>,
+        pub relative_root_turbo_json: Option<&'static [&'static str]>,
```

Comment:
We can simplify this by just storing the path instead of "components" since were' just using the first one.
```suggestion
        pub relative_root_turbo_json: Option<&'static str>,
```